requirements/base.txt

```diff
@@ -7,3 +7,5 @@ lxml>=4.4.0
 pyopenssl>=19.1.0
 furl==2.1.3
 django-sessionprofile==1.0
+django-solo
+django-simple-certmanager
```

Comment:
we should get rid of this file, installing all the requirements dependencies should be done with `pip install -e .`

That should also add python3-saml@maykin to the dependency list.

Reply:
#19